src/Misc.ino

```diff
@@ -14,6 +14,34 @@
 #include "src/Globals/Services.h"
 
 
+#ifdef ESP32
+ 
+  //MFD: adding tone support here while waiting for the Arduino Espressif implementation to catch up
+  //As recomandation is not to use external libraries the following code was taken from: https://github.com/lbernstone/Tone Thanks
+  #define TONE_CHANNEL 15
+
+  void noTone(uint8_t pin, uint8_t channel=TONE_CHANNEL)
+  {
+      ledcDetachPin(pin);
+      ledcWrite(channel, 0);
+  }
+
+  void tone(uint8_t pin, unsigned int frequency, unsigned long duration, uint8_t channel=TONE_CHANNEL)
```

Comment:
Maybe use a bit more specific function name for it?
It is only for ESP32 and `tone` is a very generic name for a function.

Reply:
We can call it toneESP32 if you want